Solutions/Corvus.Tenancy.Abstractions/Corvus/Tenancy/TenantExtensions.cs

```diff
@@ -219,11 +216,6 @@ private static string EncodeGuids(bool appendAdditionalGuid, params Guid[] guids
                 index += 16;
             }
 
-            if (appendAdditionalGuid)
-            {
```

Comment:
Interestingly, that's exactly what I did to start with. Then I changed it because I thought that the resulting call looked quite odd, with the Guid to append coming before the list of existing Guids.

Also, that boolean was only there to support the code in the `CreateChildId` method so I thought it made the code more readable to have the appending of the new Guid take place in that method rather than in `EncodeGuids`.